app/src/test/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryScreenControllerTest.kt

```diff
@@ -71,16 +69,15 @@ class NewMedicalHistoryScreenControllerTest {
     val savedPatient = PatientMocker.patient(uuid = UUID.randomUUID())
     whenever(patientRepository.saveOngoingEntryAsPatient()).thenReturn(Single.just(savedPatient))
 
-    val answersMinusNone = MedicalHistoryQuestion.values().asList()
-    val selectedAnswers = answersMinusNone.shuffled().subList(0, answersMinusNone.size / 2)
-    val unselectedAnswers = answersMinusNone.minus(selectedAnswers)
+    val questions = MedicalHistoryQuestion.values().asList()
+    val questionsAndAnswers = mutableMapOf<MedicalHistoryQuestion, MedicalHistory.Answer>()
+    questions.forEach { question ->
```

Comment:
`questions` seems to be declared for the sole purpose of generating the map of questions to answers. This can be done in a cleaner way using the kotlin collections apis like so:

```kotlin
val questionsAndAnswers = MedicalHistoryQuestion.values()
        .map { it to randomAnswer() }
        .toMap()
```